server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/ContestDownloadByID.java

```diff
@@ -60,15 +56,11 @@ public String endpoint(final Request the_request, final Response the_response) {
         result = Main.GSON.toJson(c);
       }
     } catch (final NumberFormatException e) {
-      status = HttpStatus.BAD_REQUEST_400;
-      result = "Bad contest ID";
+      invariantViolation(the_response, "Bad contest ID");
```

Comment:
Yes, it is a `400 Bad Request`.  What do you mean "Where can I find it?".  The implementation of `invariantViolation`?  It is in `AbstractEndpoint`.